Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegration.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * An integration managing a workflow for end users to report feedback via Sentry.
+ * - note: The default method to show the feedback form is via a floating widget placed in the bottom trailing corner of the screen. See the configuration classes for alternative options.
+ */
+class SentryUserFeedbackIntegration {
+    let configuration: SentryUserFeedbackConfiguration
+    private var widgetConfig: SentryUserFeedbackWidgetConfiguration?
+    private var formConfig: SentryUserFeedbackFormConfiguration?
+    private var lightThemeOverrides: SentryUserFeedbackThemeConfiguration?
+    private var darkThemeOverrides: SentryUserFeedbackThemeConfiguration?
```

Comment:
m: What will this light and dark theme be used for, since formConfig also have it?

Reply:
The properties on the form config object are the builder closures, these are the actual objects that are passed to the builders for modification and then stored until needed. See https://github.com/getsentry/sentry-cocoa/blob/d1dd7999f98b2366ab390c665d90b86148ca5cd7/Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegration.swift#L30-L39